Dockerfile

```
@@ -1,14 +1,14 @@
 FROM ubuntu:focal-20220531
 
-ARG VERSION=15.1.2
+ARG VERSION=15.2.0
 
 ENV GITLAB_VERSION=${VERSION} \
     RUBY_VERSION=2.7.6 \
     RUBY_SOURCE_SHA256SUM="e7203b0cc09442ed2c08936d483f8ac140ec1c72e37bb5c401646b7866cb5d10" \
     GOLANG_VERSION=1.17.11 \
     GITLAB_SHELL_VERSION=14.7.4 \
     GITLAB_PAGES_VERSION=1.59.0 \
```

There are updates on gitlab-shel and gitlab-pages.

```suggestion
    GITLAB_SHELL_VERSION=14.9.0 \
    GITLAB_PAGES_VERSION=1.61.0 \
```

----

note:  
During update process, we refer the files `GITALY_SERVER_VERSION`, `GITLAB_SHELL_VERSION` and `GITLAB_PAGES_VERSION` of corresponding gitlab code. 
For this case, you can check them on 
- https://gitlab.com/gitlab-org/gitlab-foss/-/blob/v15.2.0/GITALY_SERVER_VERSION
- https://gitlab.com/gitlab-org/gitlab-foss/-/blob/v15.2.0/GITLAB_SHELL_VERSION 
- https://gitlab.com/gitlab-org/gitlab-foss/-/blob/v15.2.0/GITLAB_PAGES_VERSION

note 2:  
We can set the version automatically without such settings.  
However, by providing a means to explicitly specify the version, it is possible to take measures such as downgrading when something goes wrong with the upstream-specified version. (However, if there is a problem it will be caught upstream and I'm not sure if such downgrade have been done it in this repository...)

---

I knew there was a file, but when I looked I overlooked them 😄